Configuration/UTMConfiguration.m

```diff
@@ -1327,6 +1340,7 @@ - (id)initDefaults:(NSString *)name {
         self.networkEnabled = YES;
         self.printEnabled = YES;
         self.soundEnabled = YES;
+        self.soundCardDeviceId = @0;
```

Comment:
For other values such as architectures and targets, we don't store the index but the actual value. Please do this as well. We don't want to force a mapping of id to value for future compatibility.